personal-practice/blueprint.java

```diff
@@ -0,0 +1,20 @@
+import java.awt.*;
+
+public class blueprint {
+
+    String name;
+    int ageInHumanYears;
+    Color furColor;
+    String size;
```

Comment:
I believe these fields could be defined as `private final` which would improve encapsulation.